pandas/tests/groupby/test_function.py

```diff
@@ -1060,6 +1060,55 @@ def test_size(df):
     tm.assert_series_equal(df.groupby('A').size(), out)
 
 
+# quantile
+# --------------------------------
```

Comment:
side note this file is getting pretty big, maybe should split it up a bit (later)